### ISSUE TITLE
Laravel 5.6 compilant

### ISSUE THREAD

**Issue by AntoineDewaele** [neoxia/laravel-openstack#2]
Any update for laravel 5.6 ?

**Comment by AntoineDewaele** [comment]
```
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install illuminate/contracts v5.6.5
    - Conclusion: don't install illuminate/contracts v5.6.4
    - Conclusion: remove illuminate/contracts v5.6.3
    - Conclusion: don't install illuminate/contracts v5.6.3
    - Conclusion: don't install illuminate/contracts v5.6.2
    - Conclusion: don't install illuminate/contracts v5.6.1
    - Installation request for neoxia/laravel-openstack 1.1.1 -> satisfiable by neoxia/laravel-openstack[v1.1.1].
    - Conclusion: don't install illuminate/contracts v5.6.0
    - illuminate/notifications v5.6.5 requires illuminate/contracts 5.6.* -> satisfiable by illuminate/contracts[v5.6.3, 5.6.x-dev, v5.6.0, v5.6.1, v5.6.2, v5.6.4, v5.6.5].
    - illuminate/notifications v5.6.4 requires illuminate/contracts 5.6.* -> satisfiable by illuminate/contracts[v5.6.3, 5.6.x-dev, v5.6.0, v5.6.1, v5.6.2, v5.6.4, v5.6.5].
    - illuminate/notifications v5.6.3 requires illuminate/contracts 5.6.* -> satisfiable by illuminate/contracts[v5.6.3, 5.6.x-dev, v5.6.0, v5.6.1, v5.6.2, v5.6.4, v5.6.5].
    - illuminate/notifications v5.6.2 requires illuminate/contracts 5.6.* -> satisfiable by illuminate/contracts[v5.6.3, 5.6.x-dev, v5.6.0, v5.6.1, v5.6.2, v5.6.4, v5.6.5].
    - illuminate/notifications v5.6.1 requires illuminate/contracts 5.6.* -> satisfiable by illuminate/contracts[v5.6.3, 5.6.x-dev, v5.6.0, v5.6.1, v5.6.2, v5.6.4, v5.6.5].
    - illuminate/notifications v5.6.0 requires illuminate/contracts 5.6.* -> satisfiable by illuminate/contracts[v5.6.3, 5.6.x-dev, v5.6.0, v5.6.1, v5.6.2, v5.6.4, v5.6.5].
    - illuminate/notifications 5.6.x-dev requires illuminate/contracts 5.6.* -> satisfiable by illuminate/contracts[v5.6.3, 5.6.x-dev, v5.6.0, v5.6.1, v5.6.2, v5.6.4, v5.6.5].
    - illuminate/notifications v5.6.3 requires illuminate/contracts 5.6.* -> satisfiable by illuminate/contracts[v5.6.3, 5.6.x-dev, v5.6.0, v5.6.1, v5.6.2, v5.6.4, v5.6.5].
    - Conclusion: don't install illuminate/contracts 5.6.x-dev
    - Installation request for illuminate/notifications 5.6.* -> satisfiable by illuminate/notifications[5.6.x-dev, v5.6.0, v5.6.1, v5.6.2, v5.6.3, v5.6.4, v5.6.5].```

**Comment by alexandre-butynski** [comment]
Hi @AntoineDewaele, thanks to pointing that. I will check the compatibility and publish a new release soon.

If you want to help me, you can fork the repo, update the `composer.jon` file, try it in your own project (composer doc [here](https://getcomposer.org/doc/05-repositories.md#loading-a-package-from-a-vcs-repository) to use a GitHub repository as vendor source) and tell me if everything works fine.

In addition, you would be able to continue to work on your project while I will do the boring release stuffs :)

**Comment by alexandre-butynski** [comment]
I have checked, everything should be all right with Laravel 5.6. I have published a new release (v1.1.2) that accept this version.

@AntoineDewaele please let us know if it's fine for you.
